include/ignition/rendering/Geometry.hh

```diff
@@ -66,6 +67,10 @@ namespace ignition
       /// \brief Get the material of this geometry
       /// \return Material used by this geometry
       public: virtual MaterialPtr Material() const = 0;
+
+      /// \brief Clone the geometry.
+      /// \return The cloned geometry.
+      public: virtual GeometryPtr Clone() const = 0;
```

Comment:
Note to self - I need to add tests for this method, which I plan to do once I sort out a few issues I'm having with cloning a geometry and its mesh.